.github/workflows/build.yaml

```diff
@@ -0,0 +1,279 @@
+        major="$(echo ${src_version} | cut -d. -f1)"
+        minor="$(echo ${src_version} | cut -d. -f2)"
+        patch="$(echo ${src_version} | cut -d. -f3 | cut -d- -f1)"
+
+        if ${{ inputs.release || false }}; then
+          effective_version=${major}.${minor}.${patch}
+          # hardcode to bumping "minor" for now
+          next_minor=$(( $minor + 1 ))
+          next_version="${major}.${next_minor}.${patch}-dev"
+          echo "next_version=${next_version}" >> "${GITHUB_OUTPUT}"
+          echo "next_version=${next_version}"
+        else
+          effective_version=${major}.${minor}.${patch}-${commit}
+        fi
+        echo "effective-version: ${effective_version}"
+        echo "effective_version=${effective_version}" >> "${GITHUB_OUTPUT}"
+        echo "repo_version=${src_version}" >> "${GITHUB_OUTPUT}"
+
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    needs:
+      - params
+      - version
+    strategy:
+      matrix:
+        image:
+          - name: concourse-worker
+            dockerfile: concourse-worker.dockerfile
+            oci_platforms: ${{ needs.params.outputs.oci_platforms }}
+            # todo: handle build-args
+            labels: |
+              - name: gardener.cloud/cve-categorisation
+                value:
+                  network_exposure: public
+                  authentication_enforced: true
+                  user_interaction: gardener-operator
+                  confidentiality_requirement: high
+                  integrity_requirement: high
+                  availability_requirement: high
+          - name: concourse-web
+            dockerfile: concourse-web.dockerfile
+            oci_platforms: linux/amd64
+            labels: |
+              - name: gardener.cloud/cve-categorisation
+                value:
+                  network_exposure: public
+                  authentication_enforced: true
+                  user_interaction: gardener-operator
+                  confidentiality_requirement: high
+                  integrity_requirement: high
+                  availability_requirement: high
+          - name: concourse-resource-git
+            dockerfile: concourse-resource-git.dockerfile
+            oci_platforms: ${{ needs.params.outputs.oci_platforms }}
+            labels: |
+              - name: gardener.cloud/cve-categorisation
+                value:
+                  network_exposure: protected
+                  authentication_enforced: false
+                  user_interaction: gardener-operator
+                  confidentiality_requirement: low
+                  integrity_requirement: high
+                  availability_requirement: low
+          - name: concourse-resource-time
+            dockerfile: concourse-resource-time.dockerfile
+            oci_platforms: ${{ needs.params.outputs.oci_platforms }}
+            labels: |
+              - name: gardener.cloud/cve-categorisation
```

Comment:
This categorisation changed compared to the Concourse pipeline definition. Is that on purpose?